src/steps/writeReadme/index.test.ts

```diff
@@ -334,29 +297,19 @@ describe("writeReadme", () => {
 
 			<p align=\\"center\\">
-				<a href=\\"#contributors\\" target=\\"_blank\\">
-			<!-- prettier-ignore-start -->
-			<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-			<img alt=\\"All Contributors: 2\\" src=\\"https://img.shields.io/badge/all_contributors-17-21bb42.svg\\" />
-			<!-- ALL-CONTRIBUTORS-BADGE:END -->
-			<!-- prettier-ignore-end -->
-			</a>
-				<a href=\\"https://codecov.io/gh/TestOwner/test-repository\\" target=\\"_blank\\">
-					<img alt=\\"Codecov Test Coverage\\" src=\\"https://codecov.io/gh/TestOwner/test-repository/branch/main/graph/badge.svg\\"/>
-				</a>
-				<a href=\\"https://github.com/TestOwner/test-repository/blob/main/.github/CODE_OF_CONDUCT.md\\" target=\\"_blank\\">
-					<img alt=\\"Contributor Covenant\\" src=\\"https://img.shields.io/badge/code_of_conduct-enforced-21bb42\\" />
-				</a>
-				<a href=\\"https://github.com/TestOwner/test-repository/blob/main/LICENSE.md\\" target=\\"_blank\\">
-					<img alt=\\"License: MIT\\" src=\\"https://img.shields.io/github/license/TestOwner/test-repository?color=21bb42\\">
-				</a>
-				<a href=\\"https://github.com/sponsors/TestFunding\\" target=\\"_blank\\">
-					<img alt=\\"Sponsor: On GitHub\\" src=\\"https://img.shields.io/badge/sponsor-on_github-21bb42.svg\\" />
-				</a>
+				<!-- prettier-ignore-start -->
+							<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+							<a href=\\"#contributors\\" target=\\"_blank\\"><img alt=\\"All Contributors: 2\\" src=\\"https://img.shields.io/badge/all_contributors-17-21bb42.svg\\" /></a>
+							<!-- ALL-CONTRIBUTORS-BADGE:END -->
+							<!-- prettier-ignore-end -->
+				<a href=\\"https://codecov.io/gh/TestOwner/test-repository\\" target=\\"_blank\\"><img alt=\\"Codecov Test Coverage\\" src=\\"https://codecov.io/gh/TestOwner/test-repository/branch/main/graph/badge.svg\\"/></a>
+				<a href=\\"https://github.com/TestOwner/test-repository/blob/main/.github/CODE_OF_CONDUCT.md\\" target=\\"_blank\\"><img alt=\\"Contributor Covenant\\" src=\\"https://img.shields.io/badge/code_of_conduct-enforced-21bb42\\" /></a>
+				<a href=\\"https://github.com/TestOwner/test-repository/blob/main/LICENSE.md\\" target=\\"_blank\\"><img alt=\\"License: MIT\\" src=\\"https://img.shields.io/github/license/TestOwner/test-repository?color=21bb42\\"></a>
+				<a href=\\"https://github.com/sponsors/TestFunding\\" target=\\"_blank\\"><img alt=\\"Sponsor: On GitHub\\" src=\\"https://img.shields.io/badge/sponsor-on_github-21bb42.svg\\" /></a>
 				<img alt=\\"Style: Prettier\\" src=\\"https://img.shields.io/badge/style-prettier-21bb42.svg\\" />
 				<img alt=\\"TypeScript: Strict\\" src=\\"https://img.shields.io/badge/typescript-strict-21bb42.svg\\" />
 				<img alt=\\"npm package version\\" src=\\"https://img.shields.io/npm/v/create-typescript-app?color=21bb42\\" />
-				<img alt=\\"Contributor Covenant\\" src=\\"https://img.shields.io/badge/code_of_conduct-enforced-21bb42\\" />
```

Comment:
[Praise] Nice, looks like this fixed an existing issue around duplicate Contributor Covenant badges?

Reply:
That's awesome. I'm happy for that ☺️